src/Parser/Druid/Restoration/Modules/Items/AzeriteTraits/common.js

```diff
@@ -0,0 +1,9 @@
+export function getPrimaryStatForItemLevel(itemLevel) {
```

Comment:
getPrimaryStatForItemLevel yes, but I don't know how to call AZ_SCALE_PRIMARY in stats.js
findItemLevelByPrimaryStat not really, but I guess it could be implemented in stats.js as well. no other modules uses this function right now so I figured I'd add it for resto druids only.